workflow/scripts/modify_prenetwork.py

```diff
@@ -397,4 +405,7 @@ def transmission_costs_from_modified_cost_data(n, costs, transmission, length_fa
     # change to NEP21 costs
     transmission_costs_from_modified_cost_data(n, costs_loaded, snakemake.params.transmission_costs, snakemake.params.length_factor)
 
+    if snakemake.config["must_run_biomass"]["enable"]:
```

Comment:
please pass these parameters explicitly via snakemake since they might be scenario dependent